docs/release/release_0_6_0.md

```diff
@@ -1,9 +1,11 @@
 # napari 0.6.0
 ⚠️ *Note: these release notes are still in draft while 0.6.0 is in release candidate testing.* ⚠️
 
-Friday, Mar 14, 2025 (0.6.0a1)
+Friday, Apr 11, 2025
 
 
+*Thu, Apr 10, 2025*
```

Comment:
I'm not sure why there's the two dates here?

Reply:
Oh! Because it's autogenerated from two different files and only one of them is supposed to have the date but I forgot this 😅